modules/xmpp/moderator.js

```diff
@@ -30,6 +30,8 @@ function Moderator(roomName, xmpp, emitter, settings, options) {
     this.externalAuthEnabled = false;
     this.settings = settings;
     this.options = options;
+    this.maxRetries = maxRetries || Infinity;
```

Comment:
It's not the only place that I see it in this PR but I'd like to use the example to point out that default parameters in ES6 give relatively the same functionality.